src/pages/messages/components/NviCandidateActions.tsx

```diff
@@ -161,7 +162,19 @@ export const NviCandidateActions = ({ nviCandidate, nviCandidateQueryKey }: NviC
           <>
             <Trans
               i18nKey="tasks.nvi.approve_nvi_candidate_description"
-              components={[<Typography sx={{ mb: '1rem' }} key="1" />]}
+              components={{
+                p: <Typography key="1" sx={{ mb: '1rem' }} />,
+                hyperlink: (
+                  <MuiLink
+                    key="2"
+                    href={'https://sikt.no/tjenester/nasjonalt-vitenarkiv-nva/hjelpeside-nva/NVI-rapporteringsinstruks'}
+                    sx={{ fontStyle: 'italic' }}
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  />
+                ),
+                launchIcon: <LaunchIcon key="3" fontSize="small" sx={{ ml: '0.2rem', verticalAlign: 'bottom' }} />,
```

Comment:
Trenger sikkert ikke keys lenger, nå som det ikke er et array?
```suggestion
                p: <Typography sx={{ mb: '1rem' }} />,
                hyperlink: (
                  <MuiLink
                    href={'https://sikt.no/tjenester/nasjonalt-vitenarkiv-nva/hjelpeside-nva/NVI-rapporteringsinstruks'}
                    sx={{ fontStyle: 'italic' }}
                    target="_blank"
                    rel="noopener noreferrer"
                  />
                ),
                launchIcon: <LaunchIcon fontSize="small" sx={{ ml: '0.2rem', verticalAlign: 'bottom' }} />,
```